### PR TITLE
Sample steps for compiling and start GPDB on a test box

### PR DESCRIPTION
**Pull request by ivannovick**

### CODE REVIEW

**Comment by asimrp** [comment]
Thank you Ivan!  I wish you had submitted this pull request a few weeks ago, when @GodenYao and I had to perform most of these steps by had on his CentOS VM.

:+1:

File: README.amazon_linux [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#update blockdev read ahead
+/sbin/blockdev --setra 16384 GPDB_DEVICE_NAMES
+
+# configure and build
+./configure
+make
+sudo make install
+. /usr/local/gpdb/greenplum_path.sh
+
+#python updae
```
Comment (danielgustafsson) [inline README.amazon_linux]: s/updae/upgrade/ or perhaps "update"?

**Comment by danielgustafsson** [comment]
:+1: - pushed with minor copyediting.

**Comment by ivannovick** [comment]
@danielgustafsson  thx

**Pull request closed without merging**